src/Microsoft.DotNet.MSBuildSdkResolver/Interop.NETStandard.cs

```diff
@@ -16,6 +17,16 @@ internal static partial class Interop
     {
         internal static readonly bool RunningOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
 
+        internal static string realpath (string path)
+        {
+            var buffer = new StringBuilder(4096); // PATH_MAX from linux/limits.h
```

Comment:
> We won't find dotnet if there's no dotnet available for the platform.

Doh, I didn't consider this, you're right :) I'll take a look at a nice way to do `realpath` or `readlink` then.

Btw. it looks like corefx doesn't free the string returned from realpath: [pal_io.cpp#L1279-L1283](https://github.com/dotnet/corefx/blob/4d78ca0148a7b8620bb5ede65d9e76acc81ce950/src/Native/Unix/System.Native/pal_io.cpp#L1279-L1283), [Interop.RealPath.cs#L17-L18](https://github.com/dotnet/corefx/blob/4d78ca0148a7b8620bb5ede65d9e76acc81ce950/src/Common/src/Interop/Unix/System.Native/Interop.RealPath.cs#L17-L18). Bug ?